recipes/haystac/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "haystac" %}
+{% set version = "0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: fcf5654495e86ce825d9768ac03c83d25e919d0827fc99bf3707a742c2bc848b
+
+build:
+  number: 0
+  noarch: generic
+  entry_points:
+    - haystac = haystac.cli:Haystac
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - argcomplete =1.12.1
```

Comment:
There are a lot of exact pinnings for random things like argparse and pyyaml. Are such exact dependencies really needed, or would instead things like `scipy >=1.4.1` suffice`?

Reply:
Thank you for your input ! I have am now allowing for a minimum versions instead of exact pinned ones. I have also updated the pip installation command. I hope these updates help. Please let me know if there is anything else that needs updating and thank you for your help !